x-pack/plugins/inference/public/chat_complete.ts

```diff
@@ -22,12 +39,18 @@ export function createChatCompleteApi({ http }: { http: HttpStart }): ChatComple
       functionCalling,
     };
 
-    return from(
-      http.post('/internal/inference/chat_complete', {
-        asResponse: true,
-        rawResponse: true,
+    if (stream) {
+      return from(
+        http.post('/internal/inference/chat_complete/stream', {
+          asResponse: true,
+          rawResponse: true,
+          body: JSON.stringify(body),
```

Comment:
no-op: I know this wasn't changed, but is this how we consume a streaming endpoint in Kibana? It feels like this is something that would benefit from a little syntax sugar within Core's http service.

Reply:
Yeah that's fair. I will open an issue for Core, AFAIK we/they didn't have one yet.

Reply:
Opened https://github.com/elastic/kibana/issues/198913

Reply:
Thanks for raising this